examples/get-started/pure-js/basic/app.js

```diff
@@ -57,5 +58,6 @@ new Deck({
       getTargetColor: [200, 0, 80],
       getWidth: 1
     })
-  ]
+  ],
+  _widgets: [new FullscreenWidget()]
```

Comment:
Assume you will create a new example for effects.

Reply:
@Pessimistress suggested I add to the getting started, but I'm not sure if this is what she meant.

Reply:
You should add a separate example unless you think it's a good functionality to surface on all the examples. Each get-started flavor (pure-js, react, script) demonstrate exactly the same functionality.